client/components/SurrenderForm.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorsList"
+
+const SurrenderForm = (props) => {
+  const [message, setMessage] = useState("")
+  const [errors, setErrors] = useState({})
+  const [surrender, setSurrender] = useState({
+    personName: "",
+    phoneNumber: "",
+    email: "",
+    petName: "",
+    age: "",
+    petTypeId: "",
+    imgUrl: "",
+    vaccinationStatus: "",
+    adoptionStory: "",
+  })
+
+  const handleInputChange = (event) => {
+    setSurrender({
+      ...surrender,
+      [event.currentTarget.id]: event.currentTarget.value,
+    })
+  }
+
+  const submitHandler = (event) => {
+    event.preventDefault()
+    if (isValid()) {
+      addPetForSurrender()
+    } else {
+      console.log("Inputs not valid!")
+    }
+  }
+
+  const addPetForSurrender = async () => {
+    try {
+      const response = await fetch("/api/v1/surrenders", {
+        method: "POST",
+        headers: new Headers({
+          "Content-Type": "application/json",
+        }),
+        body: JSON.stringify(surrender)
+      })
+      if (!response.ok) {
+        const errorMessage = `${response.status} ${response.statusText}`
+        throw new Error(errorMessage)
+      }
+      const responseBody = await response.json()
+      console.log("posted succesfully", responseBody)

```

Comment:
I like seeing this because it let's me know you were checking the data along the way! We can remove it now that development on this portion is over